### ISSUE TITLE
Publish action on marketplace

### ISSUE THREAD

**Issue by korthout** [korthout/backport-action#307]
Now that [`v1`](https://github.com/korthout/backport-action/releases/tag/v1.0.0) is released and the repo has been moved to [korthout/backport-action](https://github.com/korthout/backport-action), it should now be possible to publish the action on the marketplace.

**Comment by korthout** [comment]
Apparently, the name must be unique in the Marketplace. So, we'll need to find a unique name 🤷‍♂️ 

![Screen Shot 2022-12-22 at 14 00 33](https://user-images.githubusercontent.com/3511026/209139745-9dedb867-6a9a-42af-9fa0-e1b812a9777c.png)

**Comment by korthout** [comment]
This did not work:
- #308 

>Backport - Name must be unique. Cannot match an existing action, user or organization name.

**Comment by korthout** [comment]
Okay, I settled on [`Backport merged pull requests to selected branches`](https://github.com/marketplace/actions/backport-merged-pull-requests-to-selected-branches), which is ridiculously long, but unique and at least clear. I've also selected a description that is somewhat more appealing. And `copy` as the icon, as it helps visualize the goal (to copy the changes).

[![Screen Shot 2022-12-22 at 15 42 59](https://user-images.githubusercontent.com/3511026/209158678-ae7b020c-e763-4409-b460-dbc0b3a2b156.png)](https://github.com/marketplace/actions/backport-merged-pull-requests-to-selected-branches)
